test/controllers/disk_controller_test.rb

```diff
@@ -1,24 +1,31 @@
 require "test_helper"
 require "database/setup"
 
-require "active_storage/disk_controller"
-
-class ActiveStorage::DiskControllerTest < ActionController::TestCase
+class ActiveStorage::DiskControllerTest < ActionDispatch::IntegrationTest
   setup do
     @blob = create_blob
-    @routes = Routes
-    @controller = ActiveStorage::DiskController.new
   end
 
   test "showing blob inline" do
-    get :show, params: { filename: @blob.filename, encoded_key: ActiveStorage.verifier.generate(@blob.key, expires_in: 5.minutes, purpose: :blob_key) }
-    assert_equal "inline; filename=\"#{@blob.filename}\"", @response.headers["Content-Disposition"]
+    get rails_disk_blob_url(
+      filename: "hello.txt",
+      content_type: @blob.content_type,
+      encoded_key: ActiveStorage.verifier.generate(@blob.key, expires_in: 5.minutes, purpose: :blob_key)
+    )
+
+    assert_equal "inline; filename=\"#{@blob.filename.base}\"", @response.headers["Content-Disposition"]
     assert_equal "text/plain", @response.headers["Content-Type"]
   end
 
   test "sending blob as attachment" do
-    get :show, params: { filename: @blob.filename, encoded_key: ActiveStorage.verifier.generate(@blob.key, expires_in: 5.minutes, purpose: :blob_key), disposition: :attachment }
-    assert_equal "attachment; filename=\"#{@blob.filename}\"", @response.headers["Content-Disposition"]
+    get rails_disk_blob_url(
```

Comment:
Reminds me that now this should be `rails_disk_service_url` since the DiskController no longer deals with blobs, but only with keys.